src/utils/packet.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+const IlpPacket = require('ilp-packet')
+const assert = require('assert')
+const { omitUndefined } = require('../utils')
+const base64url = require('../utils/base64url')
+const debug = require('debug')('ilp:packet')
+
+const serialize = IlpPacket.serializeIlpPayment
+const parse = (packet) => {
+  try {
+    return IlpPacket.deserializeIlpPayment(packet)
+  } catch (e) {
+    debug('error while parsing packet: ' + e.message)
+    return undefined
+  }
+}
+  
+function getFromTransfer (transfer) {
+  assert(transfer, 'transfer must be defined. got: ' + transfer)
+  assert(typeof transfer === 'object', 'got invalid transfer: ' + transfer)
+  assert(Buffer.isBuffer(transfer.ilp || transfer.data),
+    'transfer.ilp or transfer.data must be buffer')
+
+  return transfer.ilp || transfer.data
```

Comment:
Removing the support for `transfer.data`. I guess having a wrapper around the packet functionality isn't strictly required, but it's been extremely useful throughout this rewrite (switching from JSON to binary, switching from LPI versions, etc.) so I'm inclined to keep it around for the future.